onedal/cluster/dbscan.py

```diff
@@ -16,15 +16,18 @@
 
 import numpy as np
 
+import onedal._backend.dbscan.clustering as onedal_backend
```

Comment:
```suggestion
import onedal._backend.dbscan.clustering as onedal_dbscan_clustering
```

or 

```suggestion
import onedal._backend.dbscan.clustering as onedal_backend_dbscan_clustering
```

I believe this makes better the readability of the code.